project/Dependencies.scala

```diff
@@ -6,39 +6,268 @@ object Dependencies {
+      "org.fusesource" % "sigar" % "1.6.4",
+      "org.hibernate" % "hibernate-validator" % "5.2.4.Final",
+      "org.hibernate.javax.persistence" % "hibernate-jpa-2.1-api" % "1.0.0.Final",
+      "org.immutables" % "value" % "2.3.2",
+      "org.javassist" % "javassist" % "3.20.0-GA",
+      jbossLogging,
+      "org.joda" % "joda-convert" % "1.8.1",
+      "org.hamcrest" % "hamcrest-core" % "1.3",
+      "org.mindrot" % "jbcrypt" % "0.3m",
+      "org.pcollections" % "pcollections" % "2.1.2",
+      "org.reactivestreams" % "reactive-streams" % "1.0.0",
+      "org.reflections" % "reflections" % "0.9.10",
+      "org.scalactic" %% "scalactic" % ScalaTestVersion,
+      "org.scalatest" %% "scalatest" % ScalaTestVersion,
+      "org.scala-lang.modules" %% "scala-java8-compat" % ScalaJava8CompatVersion,
+      "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4",
+      scalaXml,
+      "org.scala-sbt" % "test-interface" % "1.0",
+      "org.scala-stm" %% "scala-stm" % "0.7",
+      "org.springframework" % "spring-beans" % "4.2.7.RELEASE",
+      "org.springframework" % "spring-context" % "4.2.7.RELEASE",
+      "org.springframework" % "spring-core" % "4.2.7.RELEASE",
+      "org.uncommons.maths" % "uncommons-maths" % "1.2.2a",
+      "org.xerial.snappy" % "snappy-java" % "1.1.2.6",
+      "org.yaml" % "snakeyaml" % "1.16",
+      "tyrex" % "tyrex" % "1.0.1",
+      "xerces" % "xercesImpl" % "2.11.0",
+      "xml-apis" % "xml-apis" % "1.4.01"
+
+    ) ++ libraryFamily("com.fasterxml.jackson.core", JacksonVersion)(
+      "jackson-annotations", "jackson-core", "jackson-databind"
+
+    ) ++ libraryFamily("com.fasterxml.jackson.datatype", JacksonVersion)(
+      "jackson-datatype-jdk8", "jackson-datatype-jsr310", "jackson-datatype-guava", "jackson-datatype-pcollections"
+
+    ) ++ crossLibraryFamily("com.typesafe.akka", AkkaVersion)(
+      "akka-actor", "akka-cluster", "akka-cluster-sharding", "akka-cluster-tools", "akka-multi-node-testkit",
+      "akka-persistence", "akka-persistence-query-experimental", "akka-protobuf", "akka-remote", "akka-slf4j",
+      "akka-stream", "akka-stream-testkit", "akka-testkit"
+
+    ) ++ libraryFamily("com.typesafe.play", PlayVersion)(
+      "build-link", "play-exceptions", "play-netty-utils"
+
+    ) ++ crossLibraryFamily("com.typesafe.play", PlayVersion)(
+      "play", "play-datacommons", "play-functional", "play-iteratees", "play-java", "play-jdbc", "play-jdbc-api",
+      "play-json", "play-netty-server", "play-server", "play-streams", "play-ws"
+
+    ) ++ libraryFamily("ch.qos.logback", "1.1.3")(
+      "logback-classic", "logback-core"
+
+    ) ++ libraryFamily("io.netty", NettyVersion)(
+      "netty-buffer", "netty-codec", "netty-codec-http", "netty-common", "netty-handler", "netty-transport",
+      "netty-transport-native-epoll"
+
+    ) ++ libraryFamily("org.apache.logging.log4j", "2.7")(
+      "log4j-api", "log4j-core", "log4j-slf4j-impl"
```

Comment:
Log4j 1.x and 2.x have different artifact coordinates and packages, and are essentially two totally separate libraries AFAIK.